### PR TITLE
Account description

### PR DESCRIPTION
**Pull request by nazoking**
This pull request based on #813 Added group descriptions (implements #674) by @dotboris 
And use group description as bio if account type is user.

Group Management
Group Management

Group Edit
image

Group page
image

User Management
User Management

User Register
image

Edit my profile
image

User Profile page
image

### CODE REVIEW

File: src/main/twirl/gitbucket/core/account/edit.scala.html [hunk 1]
```diff
@@ -37,6 +37,11 @@
                 <input type="text" name="url" id="url" class="form-control" value="@account.url"/>
                 <span id="error-url" class="error"></span>
               </fieldset>
+              <fieldset class="form-group">
+                <label for="description" class="strong">bio (optional):</label>
```
Comment (takezoe) [inline src/main/twirl/gitbucket/core/account/edit.scala.html]: Update to `Bio`. I would like to unite the first character of labels as uppercase.

File: src/main/twirl/gitbucket/core/account/register.scala.html [hunk 2]
```diff
@@ -33,6 +33,11 @@
             <input type="text" name="url" id="url" class="form-control" value=""/>
             <span id="error-url" class="error"></span>
           </fieldset>
+          <fieldset>
+            <label for="description" class="strong">bio (optional):</label>
```
Comment (takezoe) [inline src/main/twirl/gitbucket/core/account/register.scala.html]: Same as above.

File: src/main/twirl/gitbucket/core/admin/user.scala.html [hunk 3]
```diff
@@ -66,6 +66,13 @@
             </div>
             <input type="text" name="url" id="url" class="form-control" value="@account.map(_.url)"/>
           </fieldset>
+          <fieldset class="form-group">
+            <label class="strong">bio (Optional):</label>
```
Comment (takezoe) [inline src/main/twirl/gitbucket/core/admin/user.scala.html]: Same as above.

**Comment by takezoe** [comment]
@nazoking Great! I added few comments about spelling. Could check them?

**Comment by nazoking** [comment]
@takezoe Thank you for being a reviewer. I done it.

**Comment by dotboris** [comment]
Hey @nazoking, thanks for this. Seeing someone else take my old code, fix it up and get it merged just warms my heart. :heart:

**Comment by nazoking** [comment]
@dotboris I am sorry that the reaction is slow. thanks you for your original contribute.

**Pull request merged**